README.md

```diff
@@ -18,7 +18,7 @@ Remove a selected book.
 ## Live Demo
 [source code](https://github.com/AngomRanjan/bookstore)
```

Comment:
[OPTIONAL]
You may remove the link for `source code` as it directs us to the same page 👍🏻